devops/e2e/cloudtest/build-infra.json

```diff
@@ -13,13 +13,19 @@
         {
             "name": "linux-bash-command",
             "parameters": {
-                "command": "apt list --installed"
+                "command": "sudo apt update && sudo apt upgrade -y"
```

Comment:
>"command": "sudo apt update && sudo apt upgrade -y" [](http://example.com/codeflow?start=16&length=51)

Do we need same for the other distros using different package managers?

Reply:
We do, all of our artifact definitions do a update/upgrade to ensure that packages are up-to-date when the images are produced

Reply:
I removed the update/upgrade, just to be consistent with others